### PR TITLE
Drop support for python 2.6 and 3.3

### LINKED ISSUE #379: Drop support for Python 2.6 and 3.3 which are both EOL?
* Python 2.6 reached its [end of life](https://devguide.python.org/#branchstatus) in 2013.
* Python 3.3 reached its end of life in 2017.

### PR DESCRIPTION
**Pull request by cclauss**
Fixes #379

Also steps up from pyflakes to [flake8](http://flake8.pycqa.org) for testing.

### CODE REVIEW

**Comment by blatinier** [comment]
It's ok for me. I'll merge this in the coming days.

File: .travis.yml [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
 install:
   - pip install -U pip
-  - pip install tox
-  - pip install pyflakes
+  - pip install flake8 tox
   - sudo rm -rf /dev/shm && sudo ln -s /run/shm /dev/shm
 script:
   - tox -e $TOX_ENV
-  - python -m pyflakes.__main__ $(git ls-files | grep -E "^isso/.+.py$" | grep -v "^isso/compat.py")
+  # stop the build if there are Python syntax errors or undefined names
```
Comment (jelmer) [inline .travis.yml]: Please keep the move to flake8 as a separate change. This is different from just dropping Python2.6/3.3 support, and changes what kind of syntax errors will be fatal.

**Comment by jelmer** [comment]
Looks good to me, with the exception of the flake changes.

**Comment by cclauss** [comment]
@jelmer Thanks for the merge.  Is there something wrong with #384 ?

**Pull request merged**